djangocms_rest_api/serializers/mapping.py

```diff
@@ -1,10 +1,14 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, print_function, unicode_literals
 
-from cmsplugin_filer_image.cms_plugins import FilerImagePlugin
+from django.conf import settings
 
-from djangocms_rest_api.serializers.filerimageplugin_serializer import FilerImagePluginSerializer
 
-serializer_class_mapping = {
```

Comment:
I don't think is needed because the end goal is to avoid these pre-configurations on the api addon and instead have them on their respective apps.